src/main/java/org/jenkinsci/plugins/docker/commons/fingerprint/DockerFingerprints.java

```diff
@@ -79,8 +80,9 @@ private DockerFingerprints() {} // no instantiation
      * @return Created fingerprint or null if it is not found
      * @throws IOException Fingerprint loading error
      */
+    @SuppressFBWarnings("RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE")
     public static @CheckForNull Fingerprint of(@Nonnull String id) throws IOException {
-        final Jenkins jenkins = Jenkins.getInstance(); // should be not null
+        final Jenkins jenkins = Jenkins.getInstance(); // should be not null, but technically can be
```

Comment:
Use `getInstanceOrNull` then

Reply:
Right, or drop the null check—I see no reason for this code to be run after Jenkins shutdown so the null check should not be necessary.